Kubernetes/windows/CertificateGenerator/Program.cs

```diff
@@ -0,0 +1,367 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Net;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using Org.BouncyCastle.X509;
+using Org.BouncyCastle.Utilities;
+using Org.BouncyCastle.Math;
+using Org.BouncyCastle.Security;
+using Org.BouncyCastle.Asn1.X509;
+using Org.BouncyCastle.Crypto;
+using Org.BouncyCastle.Crypto.Generators;
+using Org.BouncyCastle.Crypto.Operators;
+using Org.BouncyCastle.Crypto.Parameters;
+using Org.BouncyCastle.Pkcs;
+using System.Collections;
+using Org.BouncyCastle.Asn1;
+using System.Security.Cryptography;
+using System.Net.Http;
+using System.Text;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using System.Threading;
+using Org.BouncyCastle.OpenSsl;
+
+namespace CertificateGenerator
+
+
+{
+    class Program
+    {
+
+        internal static class Constants
```

Comment:
do we emit the telemetry if any of the certificate generation steps failed?

Reply:
We will restart the container if the cert registration or generation fails. We dont have telemetry now